peps/pep-0730.rst

```diff
@@ -241,6 +235,44 @@ unique Framework name by using the full module name as the framework name (i.e.,
 ``foo.bar._whiz.framework``). A framework is a directory; the finder will look
 for ``_whiz.dylib`` in that directory.
 
+Compilation
+-----------
+
+The only binary format that will be supported is a dynamically-linkable
+``libPython.dylib``, packaged in an iOS-compatible framework format. While the
```

Comment:
I assume this would follow the macOS pattern of a lower case "p" and a version number? 
```
lrwxrwxr-x  1 root  admin  9 Feb  9  2023 /Library/Frameworks/Python.framework/Versions/3.12/lib/libpython3.12.dylib -> ../Python
```

If so, the other references to "libPython" should also be updated.

```suggestion
``libpython3.x.dylib``, packaged in an iOS-compatible framework format. While the
```